src/screens/ReaderScreen.tsx

```diff
@@ -1,13 +1,27 @@
 import * as React from "react";
 import { FunctionComponent } from 'react';
-import { SafeAreaView, StyleSheet } from 'react-native';
+import { Dimensions, SafeAreaView, StyleSheet } from 'react-native';
 import { ReaderScreenProps } from '../navigationTypes';
 import BookReader from "../BookReader";
 
+const windowDimensions = Dimensions.get('window');
+
 export const ReaderScreen: FunctionComponent<ReaderScreenProps> = ({ route, navigation }: ReaderScreenProps) => {
+    const [windowSize, setWindowSize] = React.useState(windowDimensions);
+
+    React.useEffect(() => {
+        const subscription = Dimensions.addEventListener(
+            "change",
+            ({ window, screen }) => {
+                setWindowSize(window);
+            }
+        );
+        return () => subscription?.remove();
+    });
```

Comment:
I'm not 100% sure, but I think this adds the event listener upon every render. I think you want to add the empty dependencies array, because it only needs to run on mount instead of every time.

Reply:
N/A if using useWindowDimensions instead.